shuup/xtheme/templates/shuup/xtheme/admin/macros.jinja

```diff
@@ -33,25 +37,39 @@
                     {% endif %}
                 </td>
                 <td>
-                    <span class="active {% if not active_theme %} hidden {% endif %}"><i class="fa fa-check text-success"></i> {% trans %}Currently active{% endtrans %}</span>
                     <button
                         name="activate"
                         value="{{ theme.identifier }}"
                         {% if on_activate != "" %} type="button" onclick="{{ on_activate }}" {% else %} type="submit" {% endif %}
                         class="btn btn-sm btn-success {% if active_theme %} hidden {% endif %}">
```

Comment:
`class="btn btn-sm btn-success {% if active_theme %} hidden {% endif %}"` to `class="btn btn-sm btn-success{% if active_theme %} hidden{% endif %}"`